server/sonar-server/src/main/java/org/sonar/server/plugins/ws/UninstallPluginsWsAction.java

```diff
@@ -66,27 +59,14 @@ public void handle(Request request, Response response) throws Exception {
     UserSession.get().checkGlobalPermission(GlobalPermissions.SYSTEM_ADMIN);
     String key = request.mandatoryParam(PARAM_KEY);
     ensurePluginIsInstalled(key);
-    pluginJarsInstaller.uninstall(key);
+    pluginRepository.uninstall(key);
     response.noContent();
   }
 
+  // FIXME should be moved to {@link ServerPluginRepository#uninstall(String)}
```

Comment:
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)This block of commented-out lines of code should be removed.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Take the required action to fix the issue indicated by this comment.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)